tests/torch/test_sanity_sample.py

```diff
@@ -860,7 +860,8 @@ def accuracy_aware_config(request):
     ids=['distributed', 'dataparallel'])
 def test_accuracy_aware_training_pipeline(accuracy_aware_config, tmp_path, multiprocessing_distributed):
     config_factory = ConfigFactory(accuracy_aware_config['nncf_config'], tmp_path / 'config.json')
-
+    tmp_path = tmp_path / 'accuracy_aware'
```

Comment:
IMHO, it's not a good practice to override input argument of the function, moreover fixture one.
why not to create a local variable for it?

Reply:
Sure